file-service-persistence/src/main/java/uk/gov/justice/services/fileservice/repository/FileStore.java

```diff
@@ -31,29 +31,37 @@
     @Inject
     MetadataJdbcRepository metadataJdbcRepository;
 
+    @Inject
+    ConnectionProvider connectionProvider;
+
+    @Inject
+    NewIdProvider newIdProvider;
```

Comment:
Is there a specific need for this kind of wrapper? - random uuid should suffice?